opm/parser/eclipse/EclipseState/Eclipse3DProperties.cpp

```diff
@@ -854,7 +853,17 @@ namespace Opm {
         }
     }
 
-    // static member function
+    
+    const MessageContainer& Eclipse3DProperties::getMessageContainer() const {
```

Comment:
I don't have a good solution for this ... perhaps have a `MessageContainerManager` (`LogManager`) where you can ask for a `MessageContainer` whenever you need one?